app/views/mentor/knowledges/edit.html.haml

```diff
@@ -0,0 +1,4 @@
+= form_for ['mentor', @knowledge] do |f|
+  = f.text_field :title, placeholder: "title"
```

Comment:
__[HAML-Lint]__

Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.


<!-- Comment by [SideCI](https://sideci.com) -->